lib/Twinkly.js

```diff
@@ -36,7 +36,11 @@ class Twinkly {
         return this.requestService.get("gestalt", false)
             .then(json => {
                 this.name = json.device_name;
-                this.uuid = json.uuid;
+                if (json.uuid == "00000000-0000-0000-0000-000000000000") {
+                    this.uuid = `00000000-0000-1000-8000-${json.mac.replaceAll(":", "")}`;
```

Comment:
Please add `.toUpper()` to prettify the output as it is in the twinkly controllers